test/functionalTest/cases/4726_kafka_notif_override_fiware_service_and_servicepath/s_custom_subscription_modification.test

```diff
@@ -0,0 +1,136 @@
+# Copyright 2022 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Fiware-Service In Custom Notifications With Modification
+
+--SHELL-INIT--
+dbInit CB
+kafkaCreateTopics "${KAFKA_BOOTSTRAP_A}" sub1
+brokerStart CB 164 IPV4
+accumulatorStart --pretty-print --bootstrapServers "${KAFKA_BOOTSTRAP_A}" --kafkaTopic "^sub.*"
+
+--SHELL--
+
+#
+#
+# 01. Make accumulator subscribe to changes in E/A using custom notifications with Fiware-Service cucu
+# 02. Create E/A, using Fiware-Service fgm
+# 03. Check that the Fiware-Service cucu (and not fgm) was propagated to accumulator
+#
+
+echo "01. Make accumulator subscribe to changes in E/A using custom notifications with Fiware-Service cucu"
+echo "======================================================================================================"
+payload='{
+  "subject": {
+    "entities": [
+      {
+          "idPattern": ".*",
+          "type": "T"
+      }
+    ],
+    "condition": {
+      "attrs": []
+    }
+  },
+  "notification": {
+    "kafkaCustom": {
+          "url": "kafka://localhost:9092,localhost:9094",
+          "topic": "sub1",
+          "payload": "{ %22A%22: %22${A}%22 }",
+          "headers":  {"fiware-service": "cucu"}
```

Comment:
Test only include `fiware-service` and `fiware-servicepath` cases.

It would be great to include also a .test including a not-special header (i.e. a "random" header invented by the user).